install/install.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+if [ "$#" -ne 1 ]; then
+    echo "Invalid number of parameters"
+		exit 1
+fi
+
+JSONPATH1="{.spec.replicas}"
+JSONPATH2="{.status.readyReplicas}"
+ZkOprName=zoo-op
+ZkName=zoo
```

Comment:
These values should not be hard-coded. Users are expected to provide name for zookeeper cluster, bookkeeper cluster and pravega cluster inside the Chart - "values.yaml" file. Can we not pick the values from there?